manifests/u/UniversalMediaServer/UniversalMediaServer/10.12.0/UniversalMediaServer.UniversalMediaServer.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: UniversalMediaServer.UniversalMediaServer
+PackageVersion: 10.12.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/WowUp/WowUp/releases/download/v2.4.5/WowUp-Setup-2.4.5.exe
```

Comment:
I believe you mixed up the URL from #28080

Reply:
.... 🤦‍♂️